### PR TITLE
Fixes #1793 logical definition Visceral muscle cell

### LINKED ISSUE #1793: [Logical def] Visceral muscle cell missing "part_of" statement
**CL term**

https://ontobee.org/ontology/CL?iri=http://purl.obolibrary.org/obo/CL_0008007

**Suggested revision of logical definition**

Add equivalence axiom reflecting textual description.

### PR DESCRIPTION
**Pull request by aleixpuigb**
Fixes #1793 logical definition Visceral muscle cell

### CODE REVIEW

File: src/ontology/imports/merged_import.owl
Comment (bvarner-ebi) [inline src/ontology/imports/merged_import.owl:1]: @aleixpuigb, if you merge in the master branch which should now contain the recently refreshed import, I think that will cancel at least the ^^xsd:string changes in this file.

Comment (aleixpuigb) [reply src/ontology/imports/merged_import.owl:1]: Thank you! I have done the same with the other PR with imports.

**Review by bvarner-ebi — APPROVED** [review]
Consider adding full stop to end of text def.

The change appears to address the linked issue.

**Comment by aleixpuigb** [comment]
Thank you for your comments, @bvarner-ebi

**Pull request merged**